src/pages/Home/styles.scss

```diff
@@ -40,6 +40,22 @@
     flex-direction: row;
     width: 100%;
     flex: 1;
-    min-width: 368px;
+    min-width: calc(33.33% - 16px);
+    max-width: calc(33.33% - 16px);
+}
+
+@media (max-width: $breakpoint-md) {
+
+    .episode-list {
+        max-width: calc(50% - 12px);
+        min-width: calc(50% - 12px);
```

Comment:
Same goes here: You can use `flex-basis` and `gap` properties instead. And `min-width` and `max-width` being equal means `width = calc(33.33% - 16px)`